innovation.game.php

```diff
@@ -1390,7 +1437,10 @@ function notifyWithOnePlayerInvolved($card, $transferInfo, $progressInfo) {
             $message_for_player = clienttranslate('${You} draw and reveal ${<}${age}${>} ${<<}${name}${>>}.');
             $message_for_others = clienttranslate('${player_name} draws and reveals ${<}${age}${>} ${<<}${name}${>>}.');
             break;
-        
+        case 'deck->achievements':
+            $message_for_player = clienttranslate('${You} draw and achieve ${<}${age}${>} ${<<}${name}${>>}.');
+            $message_for_others = clienttranslate('${player_name} draws and achieves ${<}${age}${>} ${<<}${name}${>>}.');
```

Comment:
If someone draws and achieves, then they shouldn't see the name of the card.

${player_name} draws and achieves a ${<}${age}${>}.